go/client/chat_cli_utils.go

```diff
@@ -127,7 +127,7 @@ func parseConversationResolver(ctx *cli.Context, tlfName string) (resolver conve
 		return resolver, err
 	}
 	if resolver.TopicType == chat1.TopicType_CHAT && len(resolver.TopicName) != 0 {
-		return resolver, errors.New("multiple topic names are not yet supportted with chat conversations")
+		return resolver, errors.New("multiple topic names are not yet supported")
```

Comment:
hmm.. I thought it was the same but I can remove the "names" as well. Is this what you meant or did I miss something else?